src/components/AppBar/index.js

```diff
@@ -0,0 +1,23 @@
+import { AppBar, Toolbar, IconButton, Box } from '@mui/material';
+import { Logout } from '@mui/icons-material';
+
+import TaskManagementLogoLight from '../../assets/Task Management Logo - Light.png';
+
+export default () => {
+  return (
+    <AppBar position='static' sx={{ py: 1 }}>
+      <Toolbar variant='regular'>
+        <Box sx={{ width: 1, display: 'flex', justifyContent: 'center' }}>
+          <Box
+            component='img'
+            src={TaskManagementLogoLight}
+            sx={{ height: 50 }}
+          ></Box>
+        </Box>
+        <IconButton edge='start' color='inherit' aria-label='menu'>
+          <Logout fontSize='large' />
+        </IconButton>
+      </Toolbar>
+    </AppBar>
+  );
+};
```

Comment:
Menurut saya untuk setiap component di dalam folder components tidak perlu lagi dibuat folder, sebab kemungkinan besar hanya akan berisikan 1 file saja "index.js", beda dengan pages yang memerlukan banyak file di setiap page-nya.
Jadi langsung saja "components/AppBar.js".
Atau ada saran lain lagi?

Reply:
kalo menurut yang video km share semalam di sign in page itu, pakai yang "components/AppBar.js" ini kan, bagus gini sih lebih enak dibaca, aku ngikut aja dah wkwk

Reply:
Okay.

Reply:
Oh iya. index.js di folder components masih perlu? kyknya nggak deh? ya nggak sih

Reply:
Kyknya perlu? Supaya `import ... from ../../components'` aja

Reply:
> @afifabroory Kyknya perlu? Supaya `import { ... } from ../../components'` aja

Iya biar aja "components/index.js"